### ISSUE TITLE
Conflict with epics-autosave-dev

### ISSUE THREAD

**Issue by dylan171** [mdavidsaver/pyDevSup#1]
When installing epics-pydevsup-dev I get the following error:

```
devuser@develop-vmphy0:~$ sudo apt-get install epics-pydevsup-dev
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following extra packages will be installed:
  epics-pydevsup libpydevsup1.0
The following NEW packages will be installed:
  epics-pydevsup epics-pydevsup-dev libpydevsup1.0
0 upgraded, 3 newly installed, 0 to remove and 235 not upgraded.
Need to get 225 kB of archives.
After this operation, 2,222 kB of additional disk space will be used.
Do you want to continue? [Y/n] y
Get:1 http://nsclmirror.nscl.msu.edu/controls/debian/ jessie/fc1 libpydevsup1.0 amd64 1.0+0~20170327212217.1+debian08~1.gbp2fbe91 [23.1 kB]
Get:2 http://nsclmirror.nscl.msu.edu/controls/debian/ jessie/fc1 epics-pydevsup amd64 1.0+0~20170327212217.1+debian08~1.gbp2fbe91 [78.2 kB]
Get:3 http://nsclmirror.nscl.msu.edu/controls/debian/ jessie/fc1 epics-pydevsup-dev amd64 1.0+0~20170327212217.1+debian08~1.gbp2fbe91 [123 kB]
Fetched 225 kB in 0s (1,542 kB/s)            
Selecting previously unselected package libpydevsup1.0.
(Reading database ... 202845 files and directories currently installed.)
Preparing to unpack .../libpydevsup1.0_1.0+0~20170327212217.1+debian08~1.gbp2fbe91_amd64.deb ...
Unpacking libpydevsup1.0 (1.0+0~20170327212217.1+debian08~1.gbp2fbe91) ...
Selecting previously unselected package epics-pydevsup.
Preparing to unpack .../epics-pydevsup_1.0+0~20170327212217.1+debian08~1.gbp2fbe91_amd64.deb ...
Unpacking epics-pydevsup (1.0+0~20170327212217.1+debian08~1.gbp2fbe91) ...
dpkg: error processing archive /var/cache/apt/archives/epics-pydevsup_1.0+0~20170327212217.1+debian08~1.gbp2fbe91_amd64.deb (--unpack):
 trying to overwrite '/usr/lib/epics/db/save_restoreStatus.db', which is also in package epics-autosave-dev 5.7.1-2+0~20160811225726.28+debian08~1.gbpd84a6e
Selecting previously unselected package epics-pydevsup-dev.
Preparing to unpack .../epics-pydevsup-dev_1.0+0~20170327212217.1+debian08~1.gbp2fbe91_amd64.deb ...
Unpacking epics-pydevsup-dev (1.0+0~20170327212217.1+debian08~1.gbp2fbe91) ...
Errors were encountered while processing:
 /var/cache/apt/archives/epics-pydevsup_1.0+0~20170327212217.1+debian08~1.gbp2fbe91_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```
Maybe '/usr/lib/epics/db/save_restoreStatus.db' should not be packaged, instead there should be a dependency on epics-autosave-dev?

**Comment by mdavidsaver** [comment]
How about #3 ?

**Comment by mdavidsaver** [comment]
Should be resolved with #3 (now merged).  Please confirm.

**Comment by mdavidsaver** [comment]
I'm assuming that silence is acceptance.
